### DISCUSSION TITLE
Manager fails after start with psutil error

### DISCUSSION THREAD

**Discussion by laroy-sh** [swar/Swar-Chia-Plot-Manager#485] [Q&A]
Restarting view due to psutil going stale...
Traceback (most recent call last):
  File "manager.py", line 4, in <module>
    from plotmanager.library.utilities.commands import start_manager, stop_manager, view, analyze_logs
  File "/home/farmer/Swar-Chia-Plot-Manager/plotmanager/library/utilities/commands.py", line 3, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'

although psutil is installed

Requirement already satisfied: psutil in ./venv/lib/python3.8/site-packages (5.8.0)

Ubuntu 20.04.2 LTS

**Comment by swar** [comment]
Use `pip3` and `python3`

**Reply by laroy-sh** [reply]
Didn't help. Looks like #197 
Installed psutil outside of venv - then got the error for dateutils
Installed dateutils - got another error, installed all requirements outside of venv
Now the View starts, gets one successful refresh, and fails on the second one with
Restarting view due to psutil going stale...
Traceback (most recent call last):
  File "manager.py", line 36, in <module>
    view()
  File "/home/farmer/Swar-Chia-Plot-Manager/plotmanager/library/utilities/commands.py", line 123, in view
    os.execv(sys.executable, system_args)
FileNotFoundError: [Errno 2] No such file or directory

**Reply by laroy-sh** [reply]
wow, somehow it now works.. I'm not even sure what exactly was the fix, as I've done many stuff.. but I think it was probably installing dateutils in venv - it's not in the requirements list, so it was not installed.

**Reply by deannamnguyen** [reply]
> wow, somehow it now works.. I'm not even sure what exactly was the fix, as I've done many stuff.. but I think it was probably installing dateutils in venv - it's not in the requirements list, so it was not installed.

I get the same 'psutil' issue on the first machine, get it work somehow, then 2nd machine exact same error with 'psutil' but not sure what is causing it.
